pkg/render/common/networkpolicy/networkpolicy.go

```diff
@@ -254,7 +254,7 @@ func (h *NetworkPolicyHelper) ManagerSourceEntityRule() v3.EntityRule {
 	return CreateSourceEntityRule(h.namespace("tigera-manager"), "tigera-manager")
 }
 
-const PrometheusSelector = "(app == 'prometheus' && prometheus == 'calico-node-prometheus') || (app.kubernetes.io/name == 'prometheus' && prometheus == 'calico-node-prometheus')"
```

Comment:
I'm not following this change - is this related to something in this PR or is it just a cleanup you noticed?

Reply:
The feature branch this is targeting now specifically sets our own label k8s-app=tigera-prometheus. Previously, we relied on what labels prometheus put on our pods, which are defined in the above selector. So it is a cleanup, but related to the feature branch. Maybe it could have been in a separate PR.